caas/kubernetes/provider/proxy/info.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package proxy
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/juju/errors"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	core "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+const (
+	serviceAccountSecretCADataKey    = "ca.crt"
+	serviceAccountSecretNamespaceKey = "namespace"
+	serviceAccountSecretTokenKey     = "token"
+)
+
+func GetControllerProxy(
+	name,
+	apiHost string,
+	configI core.ConfigMapInterface,
+	saI core.ServiceAccountInterface,
+	secretI core.SecretInterface,
+) (*Proxier, error) {
+	cm, err := configI.Get(context.TODO(), name, meta.GetOptions{})
+	if k8serrors.IsNotFound(err) {
+		return nil, errors.NotFoundf("no controller proxy config found for %s", name)
```

Comment:
change to "controller proxy config for %s", NotFoundf already has the not found text in the Error() string.

Reply:
Done & Done